Modules/Core/Common/include/itkMacro.h

```diff
@@ -507,17 +534,16 @@ OutputWindowDisplayDebugText(const char *);
 #  define itkDebugMacro(x) ITK_NOOP_STATEMENT
 #  define itkDebugStatement(x) ITK_NOOP_STATEMENT
 #else
-#  define itkDebugMacro(x)                                                     \
-    {                                                                          \
-      using namespace ::itk::print_helper; /* for ostream << std::vector<T> */ \
-      if (this->GetDebug() && ::itk::Object::GetGlobalWarningDisplay())        \
-      {                                                                        \
-        std::ostringstream itkmsg;                                             \
-        itkmsg << "Debug: In " __FILE__ ", line " << __LINE__ << '\n'          \
-               << this->GetNameOfClass() << " (" << this << "): " x << "\n\n"; \
-        ::itk::OutputWindowDisplayDebugText(itkmsg.str().c_str());             \
-      }                                                                        \
-    }                                                                          \
+#  define itkDebugMacro(x)                                                                                           \
+    {                                                                                                                \
+      using namespace ::itk::print_helper; /* for ostream << std::vector<T> */                                       \
+      if (this->GetDebug() && ::itk::Object::GetGlobalWarningDisplay())                                              \
+      {                                                                                                              \
+        std::ostringstream itkmsg;                                                                                   \
+        itkmsg << "" x;                                                                                              \
```

Comment:
Is `""` necessary?

Reply:
Yes it is, to support macro parameters that start with a string or a output operator<<.